.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: elixir
+
+elixir: '1.5'
+
```

Comment:
Do we want to do the (now) usual `branches: {only: [develop, master...]}`?

Reply:
I don't think that's necessary unless the builds are expensive. Having branch builds is useful if we're pushing experimental stuff that doesn't have a PR yet or if `develop` diverges and a PR can't be cleanly merged.